### PR TITLE
Added link for non overridable keyboard shortcut

### LINKED ISSUE #778: Put link to non-overridable keyboard shortcuts in the settings
[This page](https://support.mozilla.org/en-US/kb/keyboard-shortcuts-perform-firefox-tasks-quickly) shows what shortcuts you can't use (Firefox won't stop you, but the shortcut won't work).

We could add a link to this to the settings page (leaving it up to the user to figure out their new shortcut).

### PR DESCRIPTION
**Pull request by JanviMahajan14**
Fixes #778

### CODE REVIEW

File: extension/options/optionsView.jsx [hunk 1]
```diff
@@ -241,6 +241,17 @@ const KeyboardShortcutSettings = ({
...
+          <p>
+            Non overridable keyboard shortcut (
+              <a
+            href="https://support.mozilla.org/en-US/kb/keyboard-shortcuts-perform-firefox-tasks-quickly"
+            target="_blank"
+            rel="noopener"
+              >
+                details
+              </a>
+            ):
```
Comment (ianb) [inline extension/options/optionsView.jsx]: I think this could be like:

```html
Note: <a href="..." ...>some keyboard shortcuts cannot be overridden</a>
```

**Comment by JanviMahajan14** [comment]
@ianb i have updated the changes suggested by you. Please have a look.

**Comment by ianb** [comment]
@JanviMahajan14 The text needs to be inside the `<a>...</a>` – as it is now there's nothing to click on.

Also be sure to check with `npm test` – right now there's a trailing whitespace problem.

**Comment by JanviMahajan14** [comment]
Sorry for my mistake. I will update this.

**Comment by JanviMahajan14** [comment]
I have to close this PR as i  messed around with git.
